### PR TITLE
Expand copyT2T operation tests to cover 1D and 3D textures.

### PR DESCRIPTION
**Pull request by Kangz**
Issue: #872 #874 #894

<hr>

**Requirements for PR author:**

- [X] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [X] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [X] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [ ] Tests are properly located in the test tree.
- [ ] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [ ] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [ ] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

File: src/webgpu/api/operation/command_buffer/image_copy.spec.ts [hunk 1]
```diff
@@ -26,7 +26,6 @@ export const description = `writeTexture + copyBufferToTexture + copyTextureToBu
 * TODO:
   - add another initMethod which renders the texture [3]
   - test copyT2B with buffer size not divisible by 4 (not done because expectContents 4-byte alignment)
-  - add tests for 1d / 3d textures
```
Comment (austinEng) [inline src/webgpu/api/operation/command_buffer/image_copy.spec.ts:29]: I think this file is still just 2d, unless we've decided to not test 1d/3d here. Or maybe you have the modifications locally and accidentally added this line to the commit.

File: src/webgpu/capability_info.ts [hunk 2]
```diff
@@ -375,8 +375,8 @@ const kDepthStencilFormatCapabilityInBufferTextureCopy = {
 
   // kSizedDepthStencilFormats
   depth16unorm: {
-    CopyB2T: ['all', 'depth-only'],
-    CopyT2B: ['all', 'depth-only'],
+    CopyB2T: [],
```
Comment (austinEng) [inline src/webgpu/capability_info.ts:378]: why these changes?

Comment (Kangz) [reply src/webgpu/capability_info.ts:378]: This is to skip the copy tests on these formats to avoid error because Dawn doesn't implement them correctly. Reverted.

File: src/webgpu/api/operation/command_buffer/copyTextureToTexture.spec.ts [hunk 3]
```diff
@@ -664,7 +702,13 @@ g.test('color_textures,non_compressed,non_array')
       dstCopyLevel,
     } = t.params;
 
+    if (dimension === '1d') {
```
[the remark this answers is not in the document]
Comment (Kangz) [reply src/webgpu/api/operation/command_buffer/copyTextureToTexture.spec.ts:705]: Done, let me know if this is what you were thinking about.

[the remark this answers is not in the document]
Comment (austinEng) [reply src/webgpu/api/operation/command_buffer/copyTextureToTexture.spec.ts]: yea, that works

**Review by austinEng — APPROVED** [review]
